measures/BuildExistingModel/measure.rb

```diff
@@ -109,6 +110,17 @@ def arguments(model)
     arg.setDescription('If true, output the annual component loads.')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeBoolArgument('build_hpxml_only', false)
+    arg.setDisplayName('Build only the HPXML file')
+    arg.setDescription('If true, measure stops after running BuildResidentialHPXML')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('apply_defaults', false)
+    arg.setDisplayName('Write default values to hpxml')
+    arg.setDescription('Sets default values to the hpxml object')
```

Comment:
```suggestion
    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('apply_hpxml_defaults', false)
    arg.setDisplayName('Apply default values to the HPXML file')
    arg.setDescription('If true, applies OpenStudio-HPXML default values to the generated HPXML file')
```

Reply:
Just trying to make things a bit clearer. If you accept this, you'll have to update the code for `apply_defaults` => `apply_hpxml_defaults`.